rohmu/atomic_opener.py

```diff
@@ -81,7 +81,7 @@ def _atomic_opener(
     _fd_spy(fd)
     try:
         file_obj = os.fdopen(fd, mode=mode, encoding=encoding)
-    except Exception:  # pylint: disable=broad-except
+    except Exception:
```

Comment:
I will add `BLE`, but note that it does not detect any of the existing broad except statements, because of https://github.com/astral-sh/ruff/issues/2212.

Reply:
Ah, I wasn't aware of that nuance, that's disappointing. I made a comment here, explaining the use case of enforcing this strictly (regardless of logging and re-raising): https://github.com/astral-sh/ruff/issues/6421#issuecomment-1971255326